src/cargo/core/compiler/mod.rs

```diff
@@ -999,3 +987,33 @@ impl Kind {
         }
     }
 }
+
+fn json_stdout(line: &str) -> CargoResult<()> {
```

Comment:
Let's rename this to `assert_is_empty`?

Reply:
Also, are we sure that rustdoc's stdout is clear?

Reply:
I'll investigate if it is possible for it to print to stdout.